automation_tools/__init__.py

```diff
@@ -444,6 +444,41 @@ def setup_default_libvirt(bridge=None, ip_address="192.168.100.1"):
     return interface
 
 
+def setup_http_proxy():
```

Comment:
can we have `proxy_info` as a param here?

Reply:
Thanks @lpramuk for comments. I have updated the PR accordingly.